content/docs/user-guide/troubleshooting.md

```diff
@@ -53,3 +53,17 @@ Unable to detect supported link types, as the
 [cache directory](/doc/command-reference/config#cache) doesn't exist. It is
 usually created automatically by DVC commands that need it, but you can create
 it manually (e.g. `mkdir .dvc/cache`) to enable this check.
+
+## Checkout failed with `store: false` outputs {#checkout-no-store}
+
+DVC is unable to checkout any output that is not present in the
+<abbr>cache</abbr>.This can usually be solved by fetching from
+[import storage](/doc/command-reference/remote). However, If `store: false`
+is used in a `.dvc` file, `dvc pull` and `dvc fetch` won't attempt to retrieve the
+data from a DVC remote.
```

Comment:
We may need to update this title, anchor, and description a little per https://github.com/iterative/dvc/pull/4581#pullrequestreview-501492896...